libvips/foreign/magick.c

```diff
@@ -286,6 +286,18 @@ magick_ismagick( const unsigned char *bytes, size_t length )
 		GetImageMagick( bytes, length, format ) );
 }
 
+int
+magick_quantize_images( Image *images,
+	const size_t depth, ExceptionInfo *exception )
+{
+	QuantizeInfo info;
+
+	GetQuantizeInfo( &info );
+	info.number_colors = (1 << depth);
+	QuantizeImages( &info, images, exception);
```

Comment:
libvips uses spaces inside brackets for function calls, so an extra space at the end.